api/src/docker/serverless.yml

```diff
@@ -0,0 +1,162 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: cad-lambdas
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+plugins:
+  - serverless-offline
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  lambdaHashingVersion: 20201221
+  ecr:
+    # In this section you can define images that will be built locally and uploaded to ECR
+    images:
+      openscadimage:
+        path: ./openscad/
+  apiGateway:
+    metrics: true
+    binaryMediaTypes:
+      - '*/*'
```

Comment:
There's a bit of nuance here that I should capture in a comment. TODO

Reply:
This `*/*` issue is a good future in-the-weeds task for someone really looking to learn lambdas!